lib/matplotlib/backend_tools.py

```diff
@@ -318,7 +321,12 @@ class ToolQuit(ToolBase):
     default_keymap = rcParams['keymap.quit']
 
     def trigger(self, sender, event, data=None):
-        Gcf.destroy_fig(self.figure)
+        try:
+            manager = self.figure.canvas.manager
+        except:
```

Comment:
Yes (I think so), but as I only get attributes it shouldn't throw any other kind of error right?  I will have to double check that other types of errors thrown inside a property decorated method still only throw Attribute Errors, e.g. key errors...